runtime-kernel/broadcom-wl/autobuild/build

```diff
@@ -0,0 +1,14 @@
+local dest="${PKGDIR}/usr/src/broadcom-wl-${PKGVER}"
```

Comment:
Please let me know if anything else is required.
I could do more shellcheck-style improvements but then it would diverge from the style of other kernel module packages.

Reply:
> Please let me know if anything else is required. I could do more shellcheck-style improvements but then it would diverge from the style of other kernel module packages.

You are welcome to go ahead with that and we can follow suit.